wrappers/rust/icicle-runtime/Cargo.toml

```diff
@@ -9,10 +9,16 @@ repository.workspace = true
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+once_cell = "1.10.0" # how to have it in dev-deps?
 
 [build-dependencies]
 cmake = "0.1.50"
 
 [features]
+# Enabling backend features means building them with the ICICLE frontend.
+# Sources must be either local (under icicle/backend) or pulled during the build process.
+# Note: Backend repositories are private and not meant for user workflows. Users should install backends directly instead.
 cuda_backend = []
 pull_cuda_backend = []
+metal_backend = []
+pull_metal_backend = []
```

Comment:
Is there a way to prevent these features from being used outside of tests, examples, or debug builds?

Reply:
not that I am aware of, if you have a solution that would be great